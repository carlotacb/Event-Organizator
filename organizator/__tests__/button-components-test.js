import * as React from "react";
// eslint-disable-next-line import/no-extraneous-dependencies
import renderer from "react-test-renderer";
import Button from "../components/StyledButton";

it(`renders correctly a button`, () => {
  const tree = renderer
    .create(<Button onPress={() => {}} title="Test" />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
