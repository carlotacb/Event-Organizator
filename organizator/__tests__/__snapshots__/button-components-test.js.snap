// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly a button 1`] = `
<View
  accessibilityState={
    {
      "busy": undefined,
      "checked": undefined,
      "disabled": undefined,
      "expanded": undefined,
      "selected": undefined,
    }
  }
  accessibilityValue={
    {
      "max": undefined,
      "min": undefined,
      "now": undefined,
      "text": undefined,
    }
  }
  accessible={true}
  collapsable={false}
  focusable={true}
  onBlur={[Function]}
  onClick={[Function]}
  onFocus={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    [
      {
        "alignItems": "center",
        "backgroundColor": "darkslategray",
        "borderRadius": 20,
        "height": 55,
        "justifyContent": "center",
        "marginBottom": 10,
        "marginLeft": 10,
        "marginRight": 10,
        "marginTop": 10,
        "width": "60%",
      },
    ]
  }
>
  <Text
    style={
      [
        {
          "color": "white",
          "fontSize": 18,
          "fontWeight": "bold",
        },
      ]
    }
  >
    Test
  </Text>
</View>
`;
